Task/update spotify android libraries (#137)
brim-borium/spotify_sdk: README.md

After you are all setup you need to add the *.aar files to your Android Project as Modules. See the [Spotify Android SDK Quick Start](https://developer.spotify.com/documentation/android/quick-start/) for detailed information.

##### Installation instructions for Android Studio 4.2+

> If you are on Android Studio 4.2+ you need to manually do all the things that the option to add .jar/.aar files did via the dialogue:

Reviewer: ```suggestion
Since Android Studio 4.2 you need to manually perform these steps in order to add .jar/.aar files:
```
How about simplifying this like that?
